### ISSUE TITLE
Usage of old rules URL

### ISSUE THREAD

**Issue by BrookJeynes** [SSWConsulting/SSW.Rules.Content#7493]
Hi @tiagov8 ,
 
As per our conversation, some rules are using the old rules URL - `rules.ssw.com.au`.

- [ ] Please go through and update every occurrence and test the link still works.

Here is a list of all occurrences:
```
./rules/do-you-send-tasks-one-email-at-a-time/rule.md:As per the examples above, if you are requesting many smaller tasks under the same topic, you should group them in the same email. In this case, [number each task that you wish to be completed](https://rules.ssw.com.au/number-tasks-questions). 
./rules/do-you-send-tasks-one-email-at-a-time/rule.md:Read [Do you reply 'Done' and delete the original email?](https://rules.ssw.com.au/dones-do-you-reply-done-and-delete-the-original-email) to see the best ways to reply "Dones".
./rules/know-that-no-carriage-returns-without-line-feed/rule.md:archivedreason: 'It is superseded by https://rules.ssw.com.au/use-environment-newline-to-make-a-new-line-in-your-string '
./rules/use-emojis-in-you-channel-names/rule.md:archivedreason: duplicated of https://rules.ssw.com.au/teams-emojis
./rules/call-SysAdmins-before-formatting-company-owned-laptops/rule.md:SysAdmins follow all kinds of procedures and standards to make sure all company-owned laptops and devices are configured correctly, with the right permissions, users and applications. Refer to [Do you know the quickest way to get your Windows 10 SOE up and running?](https://rules.ssw.com.au/do-you-know-the-quickest-way-to-get-your-windows-10-soe-up-and-running) to get the best way of setting up your computer with the right applications.
./rules/dynamics-crm-install-the-dynamics-365-app-for-outlook/rule.md:      [https://rules.ssw.com.au/set-up-your-mailbox-in-crm](/set-up-your-mailbox-in-crm)
./rules/mention-when-you-make-a-pull-request-or-comment-on-github/rule.md:archivedreason: Rule has been combined to https://rules.ssw.com.au/when-you-use-@-mentions-in-a-pbi
./rules/set-up-your-mailbox-in-crm/rule.md:After this is done, you should install the Dynamics 365 App for Outlook: [https://rules.ssw.com.au/dynamics-crm-install-the-dynamics-365-app-for-outlook](/install-the-2-add-ins)
./rules/do-you-turn-off-auto-update-on-your-sharepoint-servers/rule.md:archivedreason: https://rules.ssw.com.au/do-you-turn-off-auto-update-on-your-servers
./rules/the-different-types-of-test/rule.md:* Can be performed manually or executed using an automated framework see [https://rules.ssw.com.au/microsoft-recommended-frameworks-for-automated-ui-driven-functional-testing](/microsoft-recommended-frameworks-for-automated-ui-driven-functional-testing)
./rules/do-you-timebox-approval-requests/rule.md:&lt;This is as per the rule, [https://rules.ssw.com.au/do-you-timebox-approval-requests](/do-you-timebox-approval-requests) /&gt;
./rules/do-you-know-how-to-use-balloons-in-screenshots-for-instructions/rule.md:archivedreason: Covered by https://rules.ssw.com.au/screenshots-do-you-use-balloons-instead-of-a-wall-of-text (Checked by Andrew F and Tiago A)
./rules/improve-performance-with-lazy-loading-of-media-assets/rule.md:         src="https://rules.ssw.com.au/PublishingImages/flight.jpg">
./rules/improve-performance-with-lazy-loading-of-media-assets/rule.md:         data-src="https://rules.ssw.com.au/PublishingImages/flight.jpg">
./rules/how-to-handle-errors-in-raygun/rule.md:&lt;This email is from [https://rules.ssw.com.au/how-to-handle-errors-in-raygun/](/how-to-handle-errors-in-raygun)&gt;
./rules/image-size-instagram/rule.md:archivedreason: Replaced by https://rules.ssw.com.au/image-standard-sizes-on-social-media
./rules/the-best-ide-for-react/rule.md:  Tools - Do you know the best IDE for Angular and React? - https://rules.ssw.com.au/how-to-get-your-machine-setup
./rules/maximum-row-size-for-a-table/rule.md:archivedreason: Rule no longer required - Same information can be found at https://rules.ssw.com.au/only-use-unicode-datatypes-in-special-circumstances
./rules/analyze-your-website-stats/rule.md:Read [Rules to Better Google Rankings](https&#58;//www.ssw.com.au/SSW/Standards/Rules/RulesToBetterGoogleRankings.aspxhttps&#58;//rules.ssw.com.au/rules-to-better-google-rankings).
./rules/submit-all-dates-to-sql-server-in-iso-format/rule.md:  https://rules.ssw.com.au/parameterize-all-input-to-your-database
./rules/use-the-mediator-pattern-with-cqrs/rule.md:archivedreason: Updated to https://rules.ssw.com.au/keep-business-logic-out-of-the-presentation-layer by Jason
./rules/use-the-mediator-pattern-with-cqrs/rule.md:* In WebAPIs, this encourages developers to keep logic out of controllers. The role of controllers becomes reduced to "Receive a request from the web and immediately dispatch to mediatr". This helps implement the "Thin controllers" rule:  https://rules.ssw.com.au/do-you-use-thin-controllers-fat-models-and-dumb-views. When logic is in a controller, the only way to invoke it is via web requests. Logic in a mediator handler can be invoked by any process that is able to build the appropriate request object, such as background workers, console programs or SignalR hubs
./rules/add-quality-control-to-dones/rule.md:archivedreason: merged with a duplicate rule https://rules.ssw.com.au/do-you-know-when-to-do-use-checked-by-xxx
./rules/do-you-know-to-make-sure-that-you-book-the-next-appointment-before-you-leave-the-client/rule.md:You can see the best way to book yourself in using the CRM Service Calendar here: [http://rules.ssw.com.au/Communication/RulesToBetterCRMForUsers/Pages/How-to-book-developers-for-a-project.aspx](/scheduling-do-you-know-how-to-book-developers-for-a-project)
./rules/do-you-know-how-to-structure-a-project-for-github/rule.md:As per [Do you make awesome documentation?](https://rules.ssw.com.au/do-you-review-the-documentation):
./rules/give-clients-a-warm-welcome/rule.md:- https-rules-ssw-com-au-meetings-are-you-hospitable-in-client-meetings-at-your-office
./rules/quality-do-you-make-your-templates-accessible-to-everyone-in-your-organisation/rule.md:For details on creating your own internal Nuget repository, read this rule:[https://rules.ssw.com.au/do-you-create-a-private-repository-for-reusable-internal-code](/do-you-create-a-private-repository-for-reusable-internal-code)
./rules/track-project-documents/rule.md:* No access to external content in the search feature. E.g. Can’t search rules.ssw.com.au
./rules/keep-business-logic-out-of-the-presentation-layer/rule.md:* In Web APIs, this encourages developers to keep logic out of controllers. The role of controllers becomes reduced to "Receive a request from the web and immediately dispatch to MediatR". This helps implement the "Thin controllers" rule:  https://rules.ssw.com.au/do-you-use-thin-controllers-fat-models-and-dumb-views. When logic is in a controller, the only way to invoke it is via web requests. Logic in a mediator handler can be invoked by any process that is able to build the appropriate request object, such as background workers, console programs or SignalR hubs
./rules/avoid-empty-lines-at-the-start-of-character-columns/rule.md:archivedreason: Rule no longer required - Information moved to https://rules.ssw.com.au/avoid-spaces-and-empty-lines-at-the-start-of-character-columns
./rules/graphql-when-to-use/rule.md:* For .NET, there are a few GraphQL libraries. The best known is [GraphQL.Net](https://github.com/graphql-dotnet/graphql-dotnet) which has client and server libraries. But the best library for .NET is [Hot Chocolate](https://github.com/ChilliCream/hotchocolate). Hot Chocolate, and the associated tools and libraries, are all named after desserts, which is bad – see [https://rules.ssw.com.au/follow-naming-conventions](/follow-naming-conventions), but despite this, these are the best tools for working with GraphQL in .NET.
./rules/do-you-use-email-signatures/rule.md:archivedreason: 'Merged on this rule: https://rules.ssw.com.au/great-email-signatures'
./rules/do-you-show-your-phone-number-in-skype/rule.md:- https-rules-ssw-com-au-do-you-know-to-use-skypein-to-improve-your-communication-with-clients
./rules/quality-do-you-implement-an-error-logger-that-has-notifications/rule.md:See also: [https://rules.ssw.com.au/rules-for-error-handling](/rules-for-error-handling)
./rules/do-you-know-the-quickest-way-to-get-your-windows-soe-up-and-running/rule.md:I have followed an SSW Rule: [https://rules.ssw.com.au/do-you-know-the-quickest-way-to-get-your-windows-soe-up-and-running](/do-you-know-the-quickest-way-to-get-your-windows-soe-up-and-running) and this included: 
./rules/use-css-class-form-horizontal-to-arrange-fields-and-labels/rule.md:archivedreason: "The class no-longer exists in Bootstrap 4.\nhttps://getbootstrap.com/docs/4.5/components/forms/#form-controls \n\nRelated rule: https://rules.ssw.com.au/how-to-align-your-form-labels"
./rules/answer-im-questions-in-order/rule.md:Will work for Northwind. I have 85 items and the link is https://rules.ssw.com.au/rules-to-better-im
./rules/answer-im-questions-in-order/rule.md:Here is the link: https://rules.ssw.com.au/rules-to-better-im    
./rules/answer-im-questions-in-order/rule.md:2. Here is the link: https://rules.ssw.com.au/rules-to-better-im     
./rules/rename-a-rule/rule.md:* Physical URL - e.g. https://rules.ssw.com.au/<mark>Pages/</mark>tweet-rules-you-follow<mark>.aspx</mark>
./rules/rename-a-rule/rule.md:* Friendly URL - e.g. https://rules.ssw.com.au/tweet-rules-you-follow
./rules/do-you-know-how-to-use-social-media-effectively-in-china/rule.md:There are 4 types of Official  WeChat Accounts; “Subscription Accounts”, “Service Accounts", “Enterprise Accounts" and the “Mini Program”. Each of them has unique functions and uses, which are outlined in more detail in our Rule: [https://rules.ssw.com.au/how-to-create-wechat-official-account](/how-to-create-wechat-official-account).
./rules/how-to-use-ssw-style-in-radhtmlcontrol/rule.md:<span class="ms-rteCustom-YellowBorderBox">We have a program called <a href="http://rules.ssw.com.au/WebSites/RulesToBetterWebsitesLayout"> SSW LookOut! for Outlook</a> to check for this rule.
./rules/how-to-use-ssw-style-in-radhtmlcontrol/rule.md:  <img style="border-bottom: 0px solid; border-left: 0px solid; border-top: 0px solid; border-right: 0px solid;" border="0" alt="Lookout Reply All BCC Warning" src="http://rules.ssw.com.au/WebSites/RulesToBetterWebsitesLayout" />
./rules/do-you-know-how-to-get-approval-to-book-a-flight/rule.md:Admin - follow this rule to find the best flights: [https://rules.ssw.com.au/do-you-know-how-to-book-better-flights](/do-you-know-how-to-book-better-flights)
./rules/do-you-know-how-to-get-approval-to-book-a-flight/rule.md:Get approval from your boss to make the booking by sending a purchase please email: [https://rules.ssw.com.au/do-you-know-how-to-get-approval-for-a-purchase](/do-you-know-how-to-get-approval-for-a-purchase)
./rules/do-you-have-a-dress-code/rule.md:\<This email was sent as per: [https://rules.ssw.com.au/do-you-have-a-dress-code](/do-you-have-a-dress-code)\>
./rules/use-triggers-for-denormalized-fields/rule.md:Ideally you should be using computed columns as per https://rules.ssw.com.au/use-computed-columns-rather-than-denormalized-fields
./rules/is-everyone-in-your-team-a-standards-watchdog/rule.md:**Tiny Tip:** I’d use international format on your phone number so people outside Australia can just click to dial - as per rules.ssw.com.au/great-email-signatures
./rules/httphandlers-sections-in-webconfig-must-contain-a-clear-element/rule.md:  - https-rules-ssw-com-au-do-you-know-that-httphandlers-or-httpmodules-sections-in-web-config-must-contain-a-remove-or-a-clear-element
./rules/what-metrics-to-collect-stage-3/rule.md:* [https://rules.ssw.com.au/do-you-use-the-best-exception-handling-library](/do-you-use-the-best-exception-handling-library)
./rules/what-metrics-to-collect-stage-3/rule.md:* Application Insights - https://rules.ssw.com.au/why-you-want-to-use-application-insights
./rules/how-to-create-a-rule/rule.md:This is how you create a rule at  **rules.ssw.com.au** . 
./rules/who-is-in-charge-of-keeping-the-schedule/rule.md:TIP: We also have a handy rule on this: https://rules.ssw.com.au/do-you-set-a-clear-end-time-for-breaks
./rules/use-the-brains-of-your-company/rule.md:  - https-rules-ssw-com-au-do-you-use-the-brains-of-your-company
./rules/do-you-reward-your-employees-for-doing-their-timesheets-on-time/rule.md:\<This email was sent as per the rule: [https://rules.ssw.com.au/do-you-reward-your-employees-for-doing-their-timesheets-on-time](/do-you-reward-your-employees-for-doing-their-timesheets-on-time)\>
./rules/production-do-you-use-multiple-cameras/rule.md:Camera 3 should be identical to Camera 1 if available (it is still possible to use two different cameras but this may cause some quality compromise in the form of different colours for each). An iPhone can provide the necessary shots to make a video more interesting, but only use iPhone 7 or later footage if proper lighting, exposure & composition can be achieved (Please see rule     [https://rules.ssw.com.au/production-do-you-know-the-correct-way-to-frame-your-subject](/production-do-you-know-the-correct-way-to-frame-your-subject).)
./rules/how-to-create-a-rule-category/rule.md:1. Open browser, log in to [http://rules.ssw.com.au/admin/](/admin)
./rules/when-to-hire-more-people/rule.md:\<This email was sent as per the rule: [https://rules.ssw.com.au/when-to-hire-more-people](/when-to-hire-more-people)\>
./rules/do-you-hold-regular-company-meetings/rule.md:&lt;This is as per the rule [https://rules.ssw.com.au/do-you-hold-regular-company-meetings](/do-you-hold-regular-company-meetings) &gt;
./rules/the-standard-naming-conventions-for-tests/rule.md:  https://rules.ssw.com.au/follow-naming-conventions-for-tests-and-test-projects
./rules/add-a-bot-signature-on-automated-emails/rule.md:<This email was sent as per the rule: [https://rules.ssw.com.au/do-you-have-a-dress-code](/do-you-have-a-dress-code)>
./rules/have-a-definition-of-ready/rule.md:  - https-rules-ssw-com-au-do-you-have-a-＂-definition-of-ready-＂
./rules/follow-up/rule.md:*5.*    [*http://rules.ssw.com.au*](/)
./rules/do-you-know-when-to-use-full-stops-at-the-end-of-bullet-points/rule.md:archivedreason: "Opposite to https://rules.ssw.com.au/avoid-full-stops-in-bullet-point-lists \nWrong category “Rules to Better Navigation”\nNo value\n"
```

**Comment by tiagov8** [comment]
Hi @BrookJeynes 

I believe it's all fixed now!

Note: Pending this PR review and approval: https://github.com/SSWConsulting/SSW.Rules.Content/pull/7537

Once the PR is merged...

- [ ] 1. Please rerun your search and close off this Issue if there are 0 finds
